CHANGELOG.md

```diff
@@ -1,5 +1,132 @@
 # Changelog
 
+## ✨ yew **0.20.0** *(2022-11-xx)*
+
+#### Changelog
+
+- #### 🛠 Fixes
```

Comment:
Should the features section come before the fixes section in a major release?

Reply:
It's the same for other releases so I kept it like that